virtual-organizations/GLOW.yaml

```diff
@@ -28,6 +28,10 @@ Contacts:
   VO Manager:
   - ID: 48e1c2f26dc3479a6cf9b2de7c79d654ac27b1d1
     Name: Miron Livny
```

Comment:
Separately, should this be @lmichael107 ?

Reply:
I have no idea what these fields even mean...

Reply:
We mainly use contact lists across VOs/sites to determine who to contact for various emails. Site contacts are increasingly being used to authorize token requests (i.e., CE and OS registries)